### DISCUSSION TITLE
Angular router displays 404

### DISCUSSION THREAD

**Discussion by ujjwal-kr** [vercel/vercel#4251] [General] [answered]
Can I know how I can configure the routes for single page apps ?

**Comment by coetry** [comment, marked as answer]
Hi @ujjwal-kr ! We configure single page application routing for Angular apps without any further configuration from your end :smile: .

Just build your application as you normally would and deploy it by importing your GitHub repo or by running `now --prod` from the CLI. 

I have an example that I deployed here: https://ng-routes-test.vercel-support.app/

**Reply by ujjwal-kr** [reply]
Well yes
actually I was deploying the production files directly.
Now it's all good
thanks for helping out !
